ui/web-v2/src/assets/lang/en.json

```
@@ -501,6 +501,12 @@
   "reason.reason": "Evaluation reason",
   "reason.rule": "Rule",
   "reason.target": "Target",
+  "saveChanges.addView": "Add view",
+  "saveChanges.deleteView": "Delete view",
+  "saveChanges.editView": "Edit view",
+  "saveChanges.saveChangesBeforeExiting": "If you exit without saving the changes, your modifications will not be preserved. Would you like to save before exiting? Click 'Yes' to confirm or 'No' to exit without saving.",
+  "saveChanges.setAsPreferred": "Set as Preferred",
+  "saveChanges.viewDescription": "Setting up a view will maintain all applied filters and currently visible columns.",
```

```suggestion
  "saveChanges.addShortcut": "Add shortcut",
  "saveChanges.deleteShortcut": "Delete shortcut",
  "saveChanges.editShortcut": "Edit shortcut",
  "saveChanges.saveChangesBeforeExiting": "You have unsaved search filter changes. Are you sure you want to leave this page without saving them?",
  "saveChanges.setAsPreferred": "Set as default",
  "saveChanges.shortcutDescription": "Shortcut will save all search queries and filters you have in the current view so that next time you access this page will not need to apply them every time.",
```

Also, it needsitle is missing.
Please use `Unsaved Search Filters`.